src/client.cpp

```diff
@@ -278,6 +278,8 @@ GAME_ERROR LoadGame(const char* url)
     CInputManager::Get().OpenPort(1);
     CInputManager::Get().OpenPort(2);
     CInputManager::Get().OpenPort(3);
+
+    CInputManager::Get().OpenPort(GAME_INPUT_PORT_MOUSE);
```

Comment:
Mouse is opened twice, once here and once in `CGameClient::InitializeGameplay()`. Currently, keyboard and mouse support are determined entirely by `<supports_keyboard>` and `<supports_mouse>`. Do we want to give game clients control, e.g. open/close keyboard and mouse on command, or do we want to continue forcing these open for the entire duration of gameplay?

If we want to simplify things and force keyboard and mouse open when specified in addon.xml, then keyboard and mouse should only be opened from CGameClient.

If we want to allow game clients to open/close keyboard and mouse dynamically, we should pass the value of `supports_keyboard` and `supports_mouse` to the game client in the `game_client_properties` struct, and then open them here if either are true.